index.js

```diff
@@ -58,14 +58,14 @@ module.exports = (args, flags) => {
         }
       };
 
-      await fs.writeFile(`${flags.outputDir}/hubble-data.json`, prettyJSON(mapping), fsErrorHandler);
-
       if (flags.useStyleDictionaryOutput) {
         await fs.writeFile(
           `${flags.outputDir}/hubble-style-dictionary-tokens.json`,
           prettyJSON(mapToStyleDictionaryTokens(mapping)),
           fsErrorHandler,
         );
+      } else {
```

Comment:
@Onomanatee don't think both files have to be generated if we set a flag to explicitly use SD

Reply:
True. I can't really think of any case where you'd want both files. At first I thought it might come in handy if style dictionary can't handle all outputs (such as iOS), but since these output files are dependent on project-specific config, it's probably fine.

👍